cmd/kube-controller-manager/app/mizarcontrollers.go

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright 2020 Authors of Arktos.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package app
+
+import (
+	"net/http"
+
+	"k8s.io/klog"
+	controllers "k8s.io/kubernetes/pkg/controller/mizarcontrollers"
+)
+
+const (
+	workers = 4
```

Comment:
is there a reason we choose 4 here?

Reply:
From other controllers, I see if pod/event happen frequently, the number is higher. For rarely happened controller, the number is 1. Considering pod is core object and its quantity could be huge, I think the number 4 is proper.

I made an update to don't put it in const. Because other mizar controllers are not necessary use 4.